trello_utils.py

```diff
@@ -4,24 +4,25 @@
 TRELLO_KEY = os.environ.get('TRELLO_KEY')
 TRELLO_TKN = os.environ.get('TRELLO_TKN')
```

Comment:
You are reading these env vars at import time. That means you are reading them before the tests can load the appropriate .env file and then these variables don't get updated.

I assume you ran your tests from inside an IDE which was automatically loading .env. But that means your integration tests were actually using real config when you run the tests, and running the tests from command line fails.

The simplest fix is to replace these with functions.

Reply:
@fran-man Please could you action Hugh's comment above.